css-images-3/Overview.bs

```diff
@@ -506,7 +506,7 @@ Radial Gradients: the ''radial-gradient()'' notation {#radial-gradients}
 	The radial gradient syntax is:
 
 	<pre class=prod>
-		<<radial-gradient()>> = radial-gradient( [ <radial-gradient-syntax> ] )
+		<<radial-gradient()>> = radial-gradient( [ <<radial-gradient-syntax>> ] )
 
 		<dfn>&lt;radial-gradient-syntax></dfn> =
 			<<rg-ending-shape>> || <<rg-size>> ]? [ at <<position>> ]? ,
```

Comment:
Also note the missing `[ ` before `<<rg-ending-shape>>`, also introduced by #8775.